pennylane/ops/functions/equal.py

```diff
@@ -16,9 +16,17 @@
 """
 # pylint: disable=too-many-arguments,too-many-return-statements
 import pennylane as qml
+from pennylane.operation import Operator
 
 
-def equal(op1, op2, check_interface=True, check_trainability=True, rtol=1e-5, atol=1e-9):
+def equal(
```

Comment:
We can probably move the changes to the equal function to its own PR.

Reply:
Yep, you are right. I added the changes here because I needed the `arithmetic_depth` property. I created a new [PR](https://github.com/PennyLaneAI/pennylane/pull/2877) that will be merged after this one.